spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/util/JsonToJsonPathsConverter.groovy

```diff
@@ -336,11 +338,11 @@ class JsonToJsonPathsConverter {
 	}
 
 	private JsonPaths transformToJsonPathWithValues(def json, boolean clientSide,
-			Closure parsingClosure = MapConverter.JSON_PARSING_CLOSURE) {
-		if (!json) {
```

Comment:
Before, this would short-circuit when `json` was `null` or an empty collection or map. 

Instead we want to short circuit when `json` is null or an empty collection/map and the  `includeEmptyCheck` is disabled.

Reply:
The Groovy's NullObject is a blessing and a curse at the same time ;)